src/components/pages/serverSettings/index.tsx

```diff
@@ -93,3 +93,42 @@ export default function DashboardSettings() {
     </>
   );
 }
+
+export function EnvTooltip(
+  props: React.PropsWithChildren<{
+    envVar: string;
+    data: any;
+    varKey: string;
+  }>,
+) {
+  const state = checkPropSafe(props);
+  const enabled = state !== false;
+
+  return (
+    <Tooltip
+      label={
+        enabled
+          ? `WARNING: The ${props.envVar} environment variable takes priority over this value. Currently "${state}"`
```

Comment:
i'd remove the "WARNING" text as the tooltip is already red, which signifies this and idk if ur colorblind u can just read the rest of it